cql3/util.cc

```diff
@@ -165,4 +165,13 @@ sstring rename_column_in_where_clause(const sstring_view& where_clause, column_i
     return relations_to_where_clause(expr::conjunction{std::move(new_relations)});
 }
 
+sstring insert_whitespaces_in_json_object(const sstring& json_obj) {
+    static const boost::regex re("[:,](?![ \t])"); // looks for colons and commas which are not followed by whitespace
+    static const std::string space_adder{"$& "};
+
+    std::string result;
+    boost::regex_replace(std::back_inserter(result), json_obj.begin(), json_obj.end(), re, space_adder);
+    return result;
```

Comment:
We should create the object with spaces in the first place. Regex editing is bound to fail if the contents has the wrong characters.

Reply:
The object's string was created using rjson and I think it's impossible to add those whitespaces using rjson writer (but I don't know rjosn well).

However, there is a method (already used in `describe()`) which turns `map<string, string>` into json object string, so I changed the code to use it instead of rjson